samples/Hardware.Esp32.Rmt/NeoPixelStripLowMemory/NeoPixel/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The app.config file shouldn't be here. It's added by VS on occasions and has to be deleted.